TACC/Core-CMS-Custom
# task/wp-385-386-multiple-submitter-codes-for-listing

## Overview

The purpose of this task is to allow the registrations listing page for submitter admins to see all registration ids for all submitters they are associated with.

This task also addresses broken links on the exception forms.

## Related


- [WP-385](https://jira.tacc.utexas.edu/browse/WP-385) - Allow listing for multiple submitter_codes
-  [WP-386](https://jira.tacc.utexas.edu/browse/WP-386) - Update broken links


## Changes

- Updated query to filter registrations by multiple submitter_codes
- Updated view to unpack and registrations for multiple submitter code objects
- Updated links on exception pages to new URL

## Testing

1. Go to [http://localhost:8000/register/list-registration-requests/](http://localhost:8000/register/list-registration-requests/)
2. Check DB to see what submitter_codes you're under and what the expecting listings should be
3. Verify that the right listings come up with the associated entities and contacts
4. Make sure both actions in drop down work 
5. Go to [http://localhost:8000/submissions/threshold-exception/](http://localhost:8000/submissions/threshold-exception/) and make sure link goes to [https://sph.uth.edu/research/centers/center-for-health-care-data/texas-all-payor-claims-database/index.htm](https://sph.uth.edu/research/centers/center-for-health-care-data/texas-all-payor-claims-database/index.htm)
6. Go to [http://localhost:8000/submissions/other-exception/](http://localhost:8000/submissions/other-exception/) and make sure link goes to [https://sph.uth.edu/research/centers/center-for-health-care-data/texas-all-payor-claims-database/index.htm](https://sph.uth.edu/research/centers/center-for-health-care-data/texas-all-payor-claims-database/index.htm)
